### PR TITLE
Update enable-the-report-message-add-in.md

### PR DESCRIPTION
**Pull request by andypunt**
Adding mobile support

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/enable-the-report-message-add-in.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ If you're a global administrator or an Exchange Online administrator, and Exchan
   - Outlook 2013 SP1 or later
   - Outlook 2016 for Mac
   - Outlook included with Microsoft 365 apps for Enterprise
+  - Outlook Mobile
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/enable-the-report-message-add-in.md]:
```suggestion
  - Outlook app for iOS and Android
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - please avoid using "Outlook Mobile".

File: microsoft-365/security/office-365-security/enable-the-report-message-add-in.md (hunk 1 above)
Comment (andypunt) [reply microsoft-365/security/office-365-security/enable-the-report-message-add-in.md]: Agreed. Did this commit right?

**Comment by JohanFreelancer9** [comment]
Great. Thanks, andy.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks.

**Pull request merged**